src/auth/controllers/auth.controller.ts

```diff
@@ -60,37 +53,22 @@ export class AuthController {
               return res.sendStatus(401);
             }
 
-            const refreshToken = await this.authService.generateJwtToken(
-              JwtTokenPurpose.refresh,
-              player.id,
-            );
             const authToken = await this.authService.generateJwtToken(
               JwtTokenPurpose.auth,
               player.id,
             );
-            return res.redirect(
-              `${url}?refresh_token=${refreshToken}&auth_token=${authToken}`,
-            );
+            res.cookie('auth_token', authToken, {
+              expires: add(new Date(), { days: 7 }),
```

Comment:
honestly, with cookies, we should just do 1 year. So much easier to keep logged in for years at a time.